packages/components/src/components/popover/popover.scss

```diff
@@ -61,6 +61,7 @@
             flex-grow: 2;
             text-align: left;
             line-height: 1.5;
+            pointer-events: none;
```

Comment:
@Bahar-FS , let's apply this with a modifier `prop` and `className`, in multipliers we have a use case where Popover is interact-able, so let's not apply this as a general rule.